crates/bevy_render/src/color.rs

```diff
@@ -88,6 +127,82 @@ impl Color {
             a as f32 / u8::MAX as f32,
         )
     }
+
+    fn as_nonlinear_srgb_to_linear_srgb(self) -> Color {
+        Color {
+            red: self.red.nonlinear_to_linear_srgb(),
+            green: self.green.nonlinear_to_linear_srgb(),
+            blue: self.blue.nonlinear_to_linear_srgb(),
+            alpha: self.alpha, //alpha is always linear
+        }
+    }
+
+    // non-linear-sRGB Component Getter
+    pub fn r(&self) -> f32 {
+        self.red.linear_to_nonlinear_srgb()
+    }
+
+    pub fn g(&self) -> f32 {
+        self.red.linear_to_nonlinear_srgb()
+    }
+
+    pub fn b(&self) -> f32 {
+        self.red.linear_to_nonlinear_srgb()
+    }
+
+    // linear-sRGB Component Getter
+    pub fn g_linear(&self) -> f32 {
+        self.green
+    }
+
+    pub fn r_linear(&self) -> f32 {
+        self.red
+    }
+
+    pub fn b_linear(&self) -> f32 {
+        self.blue
+    }
+
+    pub fn a_linear(&self) -> f32 {
```

Comment:
I totally get that `alpha` _is_ linear, but I think I would still prefer to make the getters/setters `a()` and `set_a()`, as these are the methods users will use "normally". This also makes us consistent with the `rgba()` constructor